influence_benchmark/RL/base_iteration.py

```diff
@@ -147,7 +160,7 @@ def _save_kwargs(self, kwargs):
 
     def setup_backends(self, agent_device, env_device, lora_path=None):
         backends = {}
-        if agent_device != env_device:
+        if self.model_names["agent"] != self.model_names["env"]:
```

Comment:
This logic seems really sketch. I think there ways in which this can break, especially once you start having `env-influence` in your model_names

Reply:
I think @carolius you're in the best place to know what this would break with the gemma models, if anything

Reply:
I think it would only be a problem if your veto model is local and different from you env and agent which are the same. This doesn't seem like something we want to do currently.